.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
```

Comment:
- [ ] Great job setting up your linters correctly, however pls kindly ensure that all your linter results are green i.e Linter should report NO errors. you can make run `npx stylelint --fix "**/*.{css,scss}"` to fix stylelint errors if you have too many of them, but try to fix them line by line also to learn best practices.
![image](https://user-images.githubusercontent.com/52934128/176257877-b83e4b10-c53a-4647-a9c3-51bdf4eb561b.png)